nasa-gibs/worldview
# WV-3084: Add 5 Level 1 NOAA-21/VIIRS layers

## Description

Fixes #WV-3084.

Added five Level 1 NOAA-21/VIIRS layers: "VIIRS_NOAA21_Brightness_Temp_BandI5_Day", "VIIRS_NOAA21_Brightness_Temp_BandI5_Night", "VIIRS_NOAA21_CorrectedReflectance_TrueColor",		"VIIRS_NOAA21_CorrectedReflectance_BandsM3-I3-M11", "VIIRS_NOAA21_CorrectedReflectance_BandsM11-I2-I1",

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?v=-308.5617576662337,-110.39265851325798,218.45790945157833,124.98893471355025&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,VIIRS_NOAA21_Brightness_Temp_BandI5_Night,VIIRS_NOAA21_Brightness_Temp_BandI5_Day,VIIRS_NOAA21_CorrectedReflectance_TrueColor,VIIRS_NOAA21_CorrectedReflectance_BandsM11-I2-I1,VIIRS_NOAA21_CorrectedReflectance_BandsM3-I3-M11&lg=true&t=2024-04-20-T16%3A47%3A27Z)
2. Check to make sure the Brightness Temperature and Corrected Reflectances layers look correct in all projections; they all have the correct layer descriptions and image previews, image snapshots works for the layers. (NOTE: the subtitle and associated Orbit track is incorrect for the True color Corrected reflectance layer, and there is a PR out to fix that)


@nasa-gibs/worldview
